src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -89,61 +91,66 @@ public override SyntaxToken VisitToken(SyntaxToken token)
                 return token;
             }
 
-            try
+            int lineBreaksBefore;
+            bool needsSeparatorBefore;
+
+            // If this starts with skipped tokens, we'll calculate the required seperator/line breaks between the previous token and the skipped tokens instead
+            bool startsWithSkippedTokens = token.LeadingTrivia is { Count: > 0 } leading && leading[0].IsKind(SyntaxKind.SkippedTokensTrivia);
```

Comment:
>leading[0].IsKind(SyntaxKind.SkippedTokensTrivia); [](http://example.com/codeflow?start=92&length=50)

nit: consider `leading[0].IsSkippedTokensTrivia` to match usage of `IsSkippedTokensTrivia` elsewhere in this file